src/auth/credentials.ts

```diff
@@ -39,6 +39,7 @@ export interface JWTInput {
   client_id?: string;
   client_secret?: string;
   refresh_token?: string;
+  quota_project?: string;
```

Comment:
eeeee, this looks correct but I think "JWTInput" is the wrong name. It looks more like "JSONCredential"

client_id, client_secret, refresh_token have nothing to do with JWTs.

Reply:
I agree this is an oddly named interface, having not spent a huge amount of time in this codebase, I find the naming a bit difficult in general :smile: ...

I think it's partially just that auth is a complex topic, with all the approaches we support.

tldr; as we put more resources on authentication, might be worth overhauling this library in the future.